server/validate/email/validate.go

```diff
@@ -41,6 +41,8 @@ type validator struct {
 	Login string `json:"login"`
 	// Password to use for SMTP authentication.
 	SenderPassword string `json:"sender_password"`
+	// Optional to use for PLAIN, LOGIN or MD5 mechanism
```

Comment:
Please mention that the default is PLAIN, something like 
`// Authentication mechanism to use, optional. One of "login", "md5", "plain" (default).`